mods/pageviews.js

```diff
@@ -146,6 +126,23 @@ Object.keys(viewCountColumnsForArticleFlat).forEach(function(k) {
     tableSchemas.articleFlat.attributes[viewCountColumnsForArticleFlat[k]] = 'int';
 });
 
+var notFoundCatcher = function(e) {
+    if (e.status === 404) {
+        throw new rbUtil.HTTPError({
```

Comment:
ah, I was trying to modify `e.description`, `e.body` is what I was missing.  Thanks, will update.